db/seeds.rb

```diff
@@ -18,40 +18,32 @@
   )
 end
 
+5.times do
+  name = Faker::Book.author
+  description = Faker::Lorem.sentence(word_count: 10)
+  Author.create!(name: name,
+                  description: description
+  )
+end
+
 50.times do
   name = Faker::Book.unique.title
   description = Faker::Lorem.sentence(word_count: 50)
   price = Faker::Commerce.price(range: 1..100.0, as_string: true)
   publish_year = Faker::Date.between(from: '2000-01-01', to: '2022-01-01')
+  book_details = []
+  (1..5).map do |i|
```

Comment:
chỗ này sao dùng map ?

Reply:
chỗ này em muốn gán các các thông số vào 1 mảng nên e dùng map ạ

Reply:
bỏ dòng 34 đi. book_details = (1..5).map{|i| {edition: "Episode #{i}",quantity: Faker::Number.between(from: 50, to: 100)}